en/deploy/README.md

```diff
@@ -130,32 +130,64 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
 
 # Setting up our blog on PythonAnywhere
 
+## Sign up for a PythonAnywhere account
+
 > **Note** You might have already created a PythonAnywhere account earlier during the install steps – if so, no need to do it again.
 
 {% include "/deploy/signup_pythonanywhere.md" %}
 
 
-## Pulling our code down on PythonAnywhere
+## Creating an API token
+
+This is something you only need to do once.  When you've signed up for PythonAnywhere, you'll be taken to your dashboard.  Find the link near the top right to your "Accounts" page, then select the tab named "API token", and hit the button that says "Create new API token".
+
+<img src="images/pythonanywhere_create_api_token.png" alt="The API token tab on the Accounts page" />
+
 
-When you've signed up for PythonAnywhere, you'll be taken to your dashboard or "Consoles" page. Choose the option to start a "Bash" console – that's the PythonAnywhere version of a console, just like the one on your computer.
+## Configuring our site on PythonAnywhere
 
-<img src="images/pythonanywhere_bash_console.png" alt="pointing at Other: Bash in Start a new Console" />
+Go back to the main PythonAnywhere Dashboard by clicking on the logo, and choose the option to start a "Bash" console – that's the PythonAnywhere version of a command line, just like the one on your computer. 
+
+<img src="images/pythonanywhere_bash_console.png" alt="Pointing at Bash in the New Console section" />
 
 > **Note** PythonAnywhere is based on Linux, so if you're on Windows, the console will look a little different from the one on your computer.
 
-Let's pull down our code from GitHub and onto PythonAnywhere by creating a "clone" of our repo. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
+Deploying a web app on PythonAnywhere involves pulling down your code from GitHub, and then configuring PythonAnywhere to recognise it and start serving it as a web application.  There are manual ways of doing it, but PythonAnywhere provide a helper tool that will do it all for you. Let's install it first:
+
+{% filename %}PythonAnywhere command-line{% endfilename %}
+```
+$ pip3.6 install --user pythonanywhere
+```
+
+That should print out some things like `Collecting pythonanywhere`, and eventually end with a line saying `Successfully installed (...) pythonanywhere- (...)`.
+
+Now we run the helper to automatically configure our app from GitHub. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ git clone https://github.com/<your-github-username>/my-first-blog.git
+$ pa_autoconfigure_django.py https://github.com/<your-github-username>/my-first-blog.git
```

Comment:
On a fresh pythonanywhere account (`dasg2`) with nothing done except for creating the API token, this failed for me:
```
22:06 ~ $ pip3.6 install --user pythonanywhere
Collecting pythonanywhere
  Downloading pythonanywhere-0.0.8.tar.gz
Requirement already satisfied: docopt in /usr/local/lib/python3.6/dist-packages (from pythonanywhere)
Requirement already satisfied: requests in /usr/local/lib/python3.6/dist-packages (from pythonanywhere)
Building wheels for collected packages: pythonanywhere
  Running setup.py bdist_wheel for pythonanywhere ... done
  Stored in directory: /home/dasg2/.cache/pip/wheels/51/a6/09/27a2fbf112b070661d9fcc722ddbb28501d822dd227d23fc4d
Successfully built pythonanywhere
Installing collected packages: pythonanywhere
Successfully installed pythonanywhere-0.0.8
22:08 ~ $ pa_autoconfigure_django.py https://github.com/das-g/my-second-blog
< Running API sanity checks >
   \
    ~<:>>>>>>>>>
< Creating web app via API >
   \
    ~<:>>>>>>>>>
Traceback (most recent call last):
  File "/home/dasg2/.local/bin/pa_autoconfigure_django.py", line 52, in <module>
    main(arguments['<git-repo-url>'], arguments['--domain'], arguments['--python'], nuke=arguments.get('--nuke'))
  File "/home/dasg2/.local/bin/pa_autoconfigure_django.py", line 33, in main
    project.create_webapp(nuke=nuke)
  File "/home/dasg2/.local/lib/python3.6/site-packages/pythonanywhere/project.py", line 28, in create_webapp
    self.webapp.create(self.python_version, self.virtualenv.path, self.project_path, nuke=nuke)
  File "/home/dasg2/.local/lib/python3.6/site-packages/pythonanywhere/api.py", line 90, in create
    raise Exception(f'PATCH to set virtualenv path via API failed, got {response}:{response.text}')
Exception: PATCH to set virtualenv path via API failed, got <Response [400]>:{"virtualenv_path":["Warning: No virtualenv detected at this path. Do you need 
to create it?"]}
22:10 ~ $
```

Reply:
> if you want to give it another go, you can either create a fresh pythonanywhere account (das3?)

:+1: Worked fine with `pythonanywhere-0.0.9` (with the same steps as above, on another brand-new pythonanywhere account `dasg3`, i.e., as instructed in the tutorial as changed by this PR)

> or do a `pip3.6 install --user --upgrade pythonanywhere`, and then use the `--nuke` flag to `pa_autoconfigure_django.py`.

I might try that later, but as long as this option isn't mentioned in the tutorial, I don't think it's relevant to this PR.

I'll also review the rest of the changed instructions in this PR, probably some time this evening.

> incidentally, i used your repo for my test, and it's just showing me the blank django "it worked" page?

Just [as expected](https://github.com/hjwp/djangogirls-tutorial/blob/b83511bb08b0aa6ea82c4102395413d62352400d/en/deploy/README.md#you-are-live)! :smiley:

> i think by the time attendees get to the deploy chapter they do have some kind of front page / urls.py entry, right?

Nope, we have them go the "deploy early, deploy often" way.
The only thing they have by then is [slightly changed project settings](https://tutorial.djangogirls.org/en/django_start_project/#changing-settings), [one model class](https://tutorial.djangogirls.org/en/django_models/#creating-a-blog-post-model) (for Posts) and [it's registration for `/admin`](https://tutorial.djangogirls.org/en/django_admin/#django-admin). (And some model instances in their local DB.)

Due to first introducing some general Internet and Web concepts and then teaching the participants some basic Python, this deploy chapter is already around halfway into the tutorial. Depending on the length of the workshop, this is often as far as participants get on-site. As the note at the top of the chapter explains, deploying in this "early" state of the project should also enable the participants to more easily finish the tutorial at home on their own, should they not be able to finish it during a workshop.

Reply:
great!  glad it's working now. and thanks for the clarification re: the state of the repo at that point. that actually reminds me - I think the current instructions are missing the `createsuperuser` step. will amend them and add that later today, so you can test them this evening.